### PR TITLE
Release notes for week of 11/19/14

### PR DESCRIPTION
**Pull request by lamagnifica**
@srpearce , @mhoeber , @catong please let me know if your teams are releasing other interesting features and fixes this week.
@rlucioni , FYI as release master.
@griffresch , please review the Coming Soon section.

### CODE REVIEW

**Comment by srpearce** [comment]
@lamagnifica Got sidetracked - will do this first thing tomorrow morning.

**Comment by lamagnifica** [comment]
This is getting more urgent, with a 1pm release deadline coming up!
@srpearce , @mhoeber , @catong please let me know if your teams are releasing other interesting features and fixes this week.
@griffresch , please review the Coming Soon section.

**Comment by catong** [comment]
@lamagnifica I've had a request from @explorerleslie to remove mention of TNL-804 (The enrollment mode for students who are pursuing a verified certificate no longer changes if a course team member uses the Instructor Dashboard to re-enroll the students. )
She is also querying whether demographic data was removed only from the legacy instructor dashboard, not the new one, and I'll try to confirm that.

**Comment by catong** [comment]
OK, to confirm, please remove mention of TNL-804, otherwise no other changes. Thanks! :+1:

**Comment by mhoeber** [comment]
:+1:

**Comment by lamagnifica** [comment]
@explorerleslie , your deletion made

File: en_us/release_notes/source/11-19-2014.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+
+**************************
+Coming Soon
+**************************
+
+A new page on the edx.org site will soon welcome visitors who are not logged in
+when they try to enroll in a course. The new page will help visitors identify
+themselves as either new or returning users, and then present fields to collect
+either registration or log in information as appropriate.
```
Comment (srpearce) [inline en_us/release_notes/source/11-19-2014.rst]: "log in" --> "login" ("login" for noun or adjective; "log in" for verb)

File: en_us/release_notes/source/11-19-2014.rst [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+* `Building and Running an edX Course`_
+
+  Click **Help** in the upper-right corner of any page in the edX Studio
+  user interface to access relevant sections of this guide. You can also
+  download the entire guide as a PDF.
+
+  Recent changes include:
+
+  * The addition of a section on `Using the Course Wiki`_.
```
Comment (srpearce) [inline en_us/release_notes/source/11-19-2014.rst]: The addition of the `Using the Course Wiki`_ section (or "The addition of a section on <link>using the course wiki<link>").

**Comment by srpearce** [comment]
Done with edit.

**Pull request merged**